tests/ztest/zexpect/src/main.cpp

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2022 Google Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * All tests are defined in the main.c file, this cpp file checks only that the API is compatible
+ * with both C and C++.
+ */
+
+#include"main.c"
```

Comment:
nit: Add space between include and source file

Reply:
Using symlink now. Moved comment to cmake file.

Reply:
Did you add the symlink to the commit?